### ISSUE TITLE
[Bug] adding assetType prop leads to warning in console

### ISSUE THREAD

**Issue by mlecoq** [colbyfayock/next-cloudinary#377]
# **Bug Report**

## **Describe the bug**

After adding `assetType` prop in `CldImage` component I have the following warning in my console 

```
 Warning: React does not recognize the `assetType` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `assettype` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    at img
    at eval (webpack-internal:///(app-pages-browser)/../../node_modules/next/dist/client/image-component.js:129:11)
    at eval (webpack-internal:///(app-pages-browser)/../../node_modules/next/dist/client/image-component.js:238:47)
    at eval (webpack-internal:///(app-pages-browser)/../../node_modules/next-cloudinary/dist/index.mjs:21:2655)
```

## **Steps To Reproduce the error**

1. Create a next projet
2. Add CldImage component
3. Set `assetType` to `Image` 
4. Check errors in the browser console (chrome in my case)

## **Expected behaviour**

No error in console

### **Your environment**

- OS: 
- Node version: 18
- Browser name and version: Chrome v120

**Comment by colbyfayock** [comment]
hey @mlecoq thanks for the issue and the PR, looks like it make sense

curious, what is the use case? video thumbnails? realizing that could be a nice win

**Comment by colbyfayock** [comment]
released in 5.16.0

**Comment by mlecoq** [comment]
> hey @mlecoq thanks for the issue and the PR, looks like it make sense
> 
> curious, what is the use case? video thumbnails? realizing that could be a nice win

yes it was for videos, thanks !
